core/cosmosdb/cache-invalidator/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+FROM scala
+
+ENV UID=1001 \
+    NOT_ROOT_USER=owuser
+
+
+# Copy app jars
+ADD build/distributions/cache-invalidator.tar /
+
+COPY init.sh /
+RUN chmod +x init.sh
+
+RUN adduser -D -u ${UID} -h /home/${NOT_ROOT_USER} -s /bin/bash ${NOT_ROOT_USER}
+
+# It is possible to run as non root if you dont need invoker capabilities out of the controller today
+#USER ${NOT_ROOT_USER}
```

Comment:
is this relevant to this dockerfile?

Reply:
Yup per [best practices][1] its always better to use non root user

[1]: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user

Reply:
Should NON_ROOT_USER be enabled here by default (not sure of any case it is required for cache-invalidator)

Reply:
Missed that. Enabled it now